config/locales/en.yml

```diff
@@ -281,7 +285,7 @@ en:
       category: Category
       region_land: Region
       h_companies_listed_below: Member companies are listed below.
-      how_to_search: Search by specifying one or more of the search parameters below.
+      how_to_search: Search for companies in search form.
```

Comment:
grammar fix: need `the`

`how_to_search: Search for companies in the search form.`

or something a little more explicit (_using..._):

`Search for companies using the search form.`